pwintools.py

```diff
@@ -542,28 +539,33 @@ def __init__(self, cmdline, flags = 0, nostdhandles = False):
         if self._create_process() != 0:
             raise(ValueError("CreateProcess failed - Invalid arguments"))
         super(Process, self).__init__(pid=self.__pid, handle=self.__phandle)
-        if not (flags & CREATE_SUSPENDED):
+        if flags == CREATE_SUSPENDED:
             self.wait_initialized()
     
     def check_initialized(self):
         is_init = False
         try: # Accessing PEB
             self.peb.modules[1]
             is_init = True
-        except:
+        except Exception as e:
+            print(e)
             pass
         if not is_init:
-            log.info("Process {:s} not initialized ...".format(self))
+            log.info("Process {0} not initialized ...".format(self))
         return is_init
     
     def wait_initialized(self):
         while not self.check_initialized():
-            time.sleep(0.05)
+            time.sleep(0.50)
                 
     def __del__(self):
+        pass
+        """self.exit(0)
         # TODO: Kill the debugger too

```

Comment:
We should expose a config option to kill or leave the process spawned by pwintools.
By default, I think we should kill all the subprocesses, let me know if you think otherwise.